src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -326,7 +326,8 @@ private void DoAccountDropdown()
 
         private void SignIn(object obj)
         {
-            AuthenticationWindow.Open();
+            var popupWindow = (PopupWindow) PopupWindow.Open(PopupWindow.PopupView.AuthenticationView);
+            popupWindow.Initialize(EntryPoint.ApplicationManager);
```

Comment:
You want to call `InitializeWindow`, not `Initialize` here. Check `ApplicationManager.cs` for how it does that for the other window. Also probably cast to BaseWindow here, or add a `IWindow` for it so we don't have to cast.